README.md

```diff
@@ -7,6 +7,8 @@ Search Insights lets you report click, conversion and view metrics using the [Al
 
 ## Getting started
 
+> Are you using Google Tag Manager in your app? We provide a [custom template](gtm) to ease the integration.
```

Comment:
did you mean to add the link here?
is `gtm` pointing to anything?

Reply:
gtm is the folder